Get all builds for pipeline graphql (#2158)
buildkite/docs: pages/apis/graphql/graphql_cookbook.md

}
```

> ### Get all builds for a given pipeline

Reviewer: It's good to keep these headings fairly short, so how about removing "given?"

```suggestion
### Get all builds for a pipeline
```

> Retrieve all of the builds for a given pipeline. This will print out a list of builds that are run on the given pipeline and return the id, number, and url of each build.

Reviewer: General tightening + adding a blank line to make it render correctly:

```suggestion
Retrieve all of the builds for a given pipeline, including each build's ID, number, and URL.

```
